decidim-participatory_processes/app/views/decidim/participatory_processes/participatory_processes/index.html.erb

```diff
@@ -14,7 +14,7 @@ edit_link(
 <% content_for :aside do %>
   <h1 class="h3 md:h1 decorator"><%= component_name %></h1>
 
-  <%= render partial: "decidim/shared/filters", locals: { filter_sections: } %>
+  <%= render partial: "decidim/shared/filters", locals: { filter_sections:, skip_to_id: "highlighted-processes" } %>
```

Comment:
What if there are no highlighted processes?

Reply:
We have changed the attribute to link to the standard results block instead of the highlighted block which is not affected by the filter